docs/pages/admin-guides/management/guides/awsoidc-integration-rds.mdx

```diff
@@ -0,0 +1,289 @@
+can search and export them using the
+[AWS Resource Groups / Tag Editor](https://console.aws.amazon.com/resource-groups/tag-editor/find-resources).
+The following tags are applied:
+
+```code
+teleport.dev/cluster      <Var name="cluster-name"/>
+teleport.dev/origin       integration_awsoidc
+teleport.dev/integration  <Var name="my-integration"/>
+```
+
+You can also search for AWS resources created by the wizard using the `aws` cli:
+
+```code
+$ aws resourcegroupstaggingapi get-resources \
+--region <Var name="us-west-1"/> \
+--tag-filters \
+Key=teleport.dev/cluster,Values=<Var name="cluster-name"/> \
+Key=teleport.dev/integration,Values=<Var name="my-integration"/> \
+Key=teleport.dev/origin,Values=integration_awsoidc
+```
+
+## How to find the ECS deployment
+
+Navigate to the ECS service in AWS: https://console.aws.amazon.com/ecs/v2/clusters
+Make sure you select the same AWS region that you selected at the start of the
+RDS enrollment wizard.
+
+The ECS cluster that the wizard created will be named after your Teleport
+cluster except with dots `.` replaced with underscores `_` and the additional
+suffix `-teleport`.
+For example if your Teleport cluster is named
+`example.teleport.sh`, then the corresponding ECS cluster will be named
+`example_teleport_sh-teleport`.
+
+Click on your ECS cluster.
+
+![Screenshot of ECS clusters overview](../../../../img/database-access/enrollment-wizard/ecs-clusters-overview@2x.png)
+
+An ECS cluster is composed of one or more ECS services.
+Click on the service named after the VPC you chose in the enrollment wizard -
+the VPC ID is used as the suffix of the corresponding ECS service name, e.g
+`database-service-vpc-123456789abcdefgh`.
+
+![Screenshot of ECS services overview](../../../../img/database-access/enrollment-wizard/ecs-services-overview@2x.png)
+
+In the ECS service overview page you should click on the "Tasks" panel to see
+the deployment tasks.
+There should be two tasks either running or pending.
+If you don't see any tasks, check the task status filter:
+
+## Troubleshooting
+
+The enrollment wizard waits for the ECS deployment tasks to start and connect to
+your Teleport cluster.
+If that does not happen, then either the ECS tasks have failed to start or they
+do not have network connectivity to your Teleport cluster.
+You should inspect the ECS deployment to troubleshoot the issue further.
+
+Most of the troubleshooting steps below assume you already know how to find
+the ECS deployment and get more information about what is happening in ECS.
+If you do not know how to find the ECS deployment, then first read about
+[how to find your ECS deployment](#how-to-find-the-ecs-deployment).
+
+### ECS tasks not running
+
+If you see that the tasks in your deployment remain in a "pending" state or have
+stopped, then you should check individual tasks to see what is blocking them
+from starting.
+
+Navigate to the ECS deployment tasks panel:
```

Comment:
FYI. ECS added this new deployment history thing recently:
https://aws.amazon.com/about-aws/whats-new/2024/11/service-versioning-deployment-history-amazon-ecs-services/

not sure if it will make debugging easier. i haven't tried myself for a bad use case scenario.

Reply:
hmm. It can show you prior deployment subnets/security groups, but that info is already available when debugging a failing deployment. I don't think it's worth mentioning in the guide